config/main.py

```diff
@@ -1348,7 +1361,23 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cach
 
         config_db.connect()
         client = config_db.get_redis_client(config_db.CONFIG_DB)
-        client.flushdb()
+        if routing_stack != "framewave":
+            client.flushdb()
+        else:
+            # On framewave all keys have to be purged one by one to generate keyspace
+            # notifications in order for the configuration from framewave
+            # to be flushed too
```

Comment:
This requirement is weird and different from other parts in sonic. Do you have more explanation in design doc?